kcidev/subcommands/results/repository.py

```diff
@@ -0,0 +1,107 @@
+import configparser
```

Comment:
maybe this can go into `libs/git-repo.py`. @aliceinwire what do you think?

Reply:
yes, I agree